### ISSUE TITLE
Context switcher does not properly handle Taskwarrior 2.6.0's read/write functionality

### ISSUE THREAD

**Issue by apkawel** [kdheepak/taskwarrior-tui#264]
Taskwarrior 2.6.0 adds read/write functionality to contexts. Essentially, a context can be defined to read data as usual, but can also be defined to automatically write certain data to a new task that is added while that context is active.

It appears that the TUI does not support this yet. In the TUI's context switcher, each context is shown as two separate contexts (one for read, and one for write).

This should be fixed so that each context has only one entry (regardless of whether it has distinct read and write definitions).

**Comment by kdheepak** [comment]
Agreed! Thanks for opening an issue here! I haven’t been keeping up with all the latest in taskwarrior.
